src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -439,13 +439,22 @@ private void SetWorkingSetLimitsCore(IntPtr? newMin, IntPtr? newMax, out IntPtr
         /// <param name="startInfo">The start info with which to start the process.</param>
         private bool StartWithCreateProcess(ProcessStartInfo startInfo)
         {
+            string arguments;
```

Comment:
Declaration of `arguments` should be moved after the big comment block so it's closer to its initialization.